src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
+type Color = 'red' | 'green' | 'blue';
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    private sideA: number,
+    private sideB: number,
+    private sideC: number,
+  ) {
+    if (sideA <= 0 || sideA <= 0 || sideC <= 0) {
+      throw new Error('Incorrect values for triangle sides');
+    }
+
+    const maxSide: number = Math.max(sideA, Math.max(sideB, sideC));
```

Comment:
```suggestion
    const maxSide: number = Math.max(sideA, sideB, sideC);
```

Reply:
I forgot that this method works with any quantity of numbers (